Git Formats/tests/syntax_test_git_commit_message

```diff
@@ -0,0 +1,136 @@
+#                  ^ punctuation.definition.reference.issue.git
+#                  ^^^^ constant.other.reference.issue.git
+#                       ^^^^^^^^^^^^^^^^^^^ - comment
+fix #203
+# <- meta.message.git.commit meta.reference.issue.git keyword.other.resolved-issue.git
+#^^^^^^^ meta.message.git.commit
+#^^^^^^^ meta.reference.issue.git
+#^^ keyword.other.resolved-issue.git
+#   ^ punctuation.definition.reference.issue.git
+#   ^^^^ constant.other.reference.issue.git
+fixes #203
+# <- meta.message.git.commit meta.reference.issue.git keyword.other.resolved-issue.git
+#^^^^^^^^^ meta.message.git.commit
+#^^^^^^^^^ meta.reference.issue.git
+#^^^^ keyword.other.resolved-issue.git
+#     ^ punctuation.definition.reference.issue.git
+#     ^^^^ constant.other.reference.issue.git
+fixed #203
+# <- meta.message.git.commit meta.reference.issue.git keyword.other.resolved-issue.git
+#^^^^^^^^^ meta.message.git.commit
+#^^^^^^^^^ meta.reference.issue.git
+#^^^^ keyword.other.resolved-issue.git
+#     ^ punctuation.definition.reference.issue.git
+#     ^^^^ constant.other.reference.issue.git
+Fix: #203
+# <- meta.message.git.commit meta.reference.issue.git keyword.other.resolved-issue.git
+#^^^^^^^^ meta.message.git.commit
+#^^^^^^^^ meta.reference.issue.git
+#^^ keyword.other.resolved-issue.git
+#  ^ punctuation.separator.key-value.resolved-issue.git
+#    ^ punctuation.definition.reference.issue.git
+#    ^^^^ constant.other.reference.issue.git
+#
+  see: https://github.com/sublimehq
+#      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.message.git.commit markup.underline.link.git
+
+  Change-Id: I3b3613e336397febf02cd8e5c14c53d493343e93
+#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.message.git.commit
+# ^^^^^^^^^ keyword.other.change-id.git.commit
+#          ^ punctuation.separator.key-value.git.commit
+#            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ constant.language.change-id.git.commit
+  Signed-off-by: username <user.name@domain.com>
+#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.message.git.commit
+#                         ^^^^^^^^^^^^^^^^^^^^^^ meta.reference.email.git
+# ^^^^^^^^^^^^^ keyword.other.signed-off-by.git.commit
+#              ^ punctuation.separator.key-value.git.commit
+#                         ^ punctuation.definition.reference.email.begin.git
+#                          ^^^^^^^^^^^^^^^^^^^^ entity.name.reference.email.git
+#                              ^ - punctuation.separator.email.git
+#                                   ^ punctuation.separator.email.git
+#                                          ^ punctuation.separator.domain.git
+#                                              ^ punctuation.definition.reference.email.end.git
+  This commit references a hash: 21dde213
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.message.git.commit
+#                                ^^^^^^^^ constant.other.hash.git
+  This commit does not references a hash: user@21dde213
```

Comment:
I suppose the "comment" on this line here is incorrect since the tests assert hash highlighting for this line.

Reply:
Those tests are just copies from syntax_test_git_commit, but yes you are right. IIRC, the related pattern has been updated as some code hosters supported it others not (at the time implementing that change).

Not a big deal though.